run.py

```diff
@@ -1,9 +1,24 @@
-from app import cli, create_app, db
+from app import cli, create_app, db, search_client, index
 from app.models import Category, Language, Resource, db
+import os
 
 app = create_app()
 cli.register(app, db)
 
+with app.app_context():
+    if os.environ.get('INDEX_CHECKED') == "FALSE":
+        query = Resource.query
+        indicies = search_client.list_indices()
+        for ind in indicies['items']:
+            if ind['name'] == 'resources_api':
+                curr_index = ind
+                if curr_index['entries'] != query.all().count():
```

Comment:
Should this actually be `if curr_index['entries'] != query.count():`? I couldn't get it to work without making that change